sdcm/utils/remote_logger.py

```diff
@@ -299,6 +300,34 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"
 
 
+class KubernetesWrongSchedulingLogger(CommandClusterLoggerBase):
+    restart_delay = 30
+    WRONG_SCHEDULED_PODS_MESSAGE = "Not allowed pods are scheduled on Scylla node found"
+
+    @property
+    def _logger_cmd(self) -> str:
+        if not self._cluster.allowed_labels_on_scylla_node:
+            return ''
+
+        wrong_scheduled_pods_on_scylla_node = []
+
+        for node in self._cluster.pools[self._cluster.SCYLLA_POOL_NAME].nodes.items:
+            for pod in KubernetesOps.list_pods(self._cluster, field_selector=f"spec.nodeName={node.metadata.name}"):
```

Comment:
Here you do separate API call per node. It is redundant, you can use multiple-values filter like here:

https://github.com/scylladb/scylla-cluster-tests/blob/be05cc50/upgrade_test.py#L1003

Reply:
done